saw-core/src/SAWCore/SharedTerm.hs

```diff
@@ -403,6 +402,12 @@ data SharedContext = SharedContext
   , scGlobalEnv      :: IORef (HashMap Ident Term)
   , scNextVarIndex   :: IORef VarIndex
   }
+-- Invariant: scGlobalEnv is a cache with one entry for every global
+-- declaration in the 'ModuleMap' whose name is a 'ModuleIdentifier'.
```

Comment:
When you say "in the `ModuleMap`", do you mean `scModuleMap` in particular? If so, worth clarifying.

Reply:
Yes, what I really meant was "in the ModuleMap pointed to by scModuleMap". I'll change it to mention scModuleMap if that's more clear.